docs/getting-started-guides/scratch.md

```diff
@@ -472,13 +472,13 @@ because of how this is used later.
 - Alternate, manual approach:
   1. Set `--configure-cbr0=false` on kubelet and restart.
   1. Create a bridge
-  - e.g. `brctl addbr cbr0`.
-  1. Set appropriate MTU
-  - `ip link set dev cbr0 mtu 1460` (NOTE: the actual value of MTU will depend on your network environment)
+     - `brctl addbr cbr0`.
+  1. Set appropriate MTU. NOTE: the actual value of MTU will depend on your network environment
+     - `ip link set dev cbr0 mtu 1460`
   1. Add the clusters network to the bridge (docker will go on other side of bridge).
-  - e.g. `ip addr add $NODE_X_BRIDGE_ADDR dev cbr0`
+     - `ip addr add $NODE_X_BRIDGE_ADDR dev cbr0`
```

Comment:
nit: Maybe say ```$YOUR_NODE_X_BRIDGE_ADDR``` or another way of telling the user that it's not some automatically created environment variable. I think most people would figure it out upon initial reading but without the e.g. for this line I could see some people misunderstanding.